modules/Hoot/managers/api.js

```diff
@@ -701,19 +701,19 @@ export default class API {
     exportDataset( data ) {
         data.tagoverrides =  JSON.stringify(
             Object.assign(data.tagoverrides || {}, {
-                'error:circular':'',
-                'hoot:building:match':'',
-                'hoot:status':'',
-                'hoot:review:members':'',
-                'hoot:review:score':'',
-                'hoot:review:note':'',
-                'hoot:review:sort_order':'',
-                'hoot:review:type':'',
-                'hoot:review:needs':'',
-                'hoot:score:match':'',
-                'hoot:score:miss':'',
-                'hoot:score:review':'',
-                'hoot:score:uuid':''
+                // 'error:circular':'',
+                // 'hoot:building:match':'',
+                // 'hoot:status':'',
+                // 'hoot:review:members':'',
+                // 'hoot:review:score':'',
+                // 'hoot:review:note':'',
+                // 'hoot:review:sort_order':'',
+                // 'hoot:review:type':'',
+                // 'hoot:review:needs':'',
+                // 'hoot:score:match':'',
+                // 'hoot:score:miss':'',
+                // 'hoot:score:review':'',
+                // 'hoot:score:uuid':''
```

Comment:
No need for default tag overrides removing hoot tags.  This is done by visitor params in the export command